plan/README.md

```diff
@@ -0,0 +1,71 @@
+
+## Goals
+
+The primary goals are of the site are given below:
+
+### Breaking Down Spending Habits
+* Helping users 
+* 
+*
+### Help Users Learn About a Zero Based Budget
+* helping users learned how to break down there pay checks 
+* 
+*
+### Promote My Professional Social Profiles
+* Allows the able to self promote 
+* Allows viewers to see other social media sites where I talk about the important of learning financial literary at a young age. 
+
+## Target User
+
+The target users are young adults who want to learn more about what to do with their money and how to save and invest it. 
+
+### Jacob - Highschool Student
+
+<img src="img/highschool_student.jpg" width=200>
+
+jacob is a 17 year-old highschool junior who just got his first job. He wants to learn how to make the most out of the small amount of money he is earning and looking for a advice for how he can make the most out of each paycheck 
+
+He is visiting my page to soak in as much knowledge as he can get and find out more ways that he can be smarter with the money he's earning. Jacob is taking the first step in learning about financial literary 
+
+### Mary - College Student
+
+<img src="img/college_student.jpg" width=200>
+
+Mary is 22 year-old college senior who's decided to move out of her parents house and live on her own for her last year of college. She had been saving most of her paychecks in prepartion for moving out. 
+
+Mary has notice the money she has saved is depleting much quicker then she thought it would and is looking for finacnal advice on how to budget her spending.
+
+She likes what my website has to say and follows me on other social media sites via links on website page to keep up with more of my daily conent on how to be more financially literate. 
+
+## Content 
+
+### Home
+
+This will be the page user see when I first arrive on the site. The home page won't contain a lot of information. It will have my name, the idea behind wanting to create the wesite, and my mission for the site along with a with a few pictures
+
+## About 
+
+Page will contain a brief narrative about myself. It will contain some information about my professional life as well. 
+
+### Budget Breakdown page
+ This page will stress the important of learn what a budget is and how it can be used to help you see where/how your money is being spent
+
+### investing Advice 
+
+This page will contain information on how to make small but smart and easy investments. The point of this page it to show young peolpe it not complicated make to  smart long term investment to further ther financal status.
+
+### Contact Informtion 
+
+Contact information will allways be availbe in the nav bar. I want the contact information to be visible to the user and easy to access. 
+
+
+
```

Comment:
Add a "similar sites" section here. You will want to provide links to the sites, one small picture of the site would be nice as well. Discuss why you have included this site for reference. Will you use any ideas from this site in the development of your project?